shared/src/main/java/com/sap/psr/vulas/shared/json/model/ExemptionBug.java

```diff
@@ -117,17 +131,42 @@ public static ExemptionSet readFromConfiguration(Configuration _cfg) {
 		final ExemptionSet exempts = new ExemptionSet();
 		
 		// New format
-		Iterator<String> iter = _cfg.subset(CFG_PREFIX).getKeys();
+		final Iterator<String> iter = _cfg.getKeys(CFG_PREFIX);
 		while(iter.hasNext()) {
 			final String k = iter.next();
-			if(!k.equals("")) {
+			if(k.endsWith("." + "reason")) {
 				final String[] key_elements = k.split("\\.");
-				final int l = key_elements.length; 
-				if(l<2) {
-					log.error("Exemption with key [" + CFG_PREFIX + "." + k + "] has less than 2 elements");
+				if(key_elements.length == 5) {
+					final String vuln = key_elements[3];
+					final String reason = _cfg.getString(CFG_PREFIX + "." + vuln + "." + "reason");
+					final String[] libs = _cfg.getStringArray(CFG_PREFIX + "." + vuln + "." + "libraries");
+					
+					if(libs==null || libs.length==0) {
+						exempts.add(new ExemptionBug(vuln, ExemptionBug.ALL, reason));
+					}
+					else {
+						for(String lib: libs) {
+							if(lib.startsWith("pkg:")) {
+								try {
+									final PackageURL purl = new PackageURL(lib);
+									if(purl.getName()==null) {
```

Comment:
This is linked to the previous comment, why not requiring that also the namespace is not null?